osutils/utils.go

```diff
@@ -0,0 +1,39 @@
+package osutils
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/rotisserie/eris"
+)
+
+var (
+	MissingEnvVarError = func(envVar string) error {
+		return fmt.Errorf("Missing %s environment variable", envVar)
+	}
+)
+
+/**
+  Returns error if the environment variable is empty
+*/
+func GetEnvE(envVar string) (string, error) {
```

Comment:
Nit: we generally prefix with `must` if an error is returned in case of a missing value

Reply:
https://www.journal.petrausch.info/post/2020/05/must-pattern/
> In conclusion, use the prefix Must if your method or function panics. Be careful if an API provides a Must function. Expect that function to panic. 

I don't want this function to panic, but return an error.